src/sentry/web/frontend/debug/debug_performance_issue.py

```diff
@@ -22,9 +22,6 @@
 
 
 class DebugPerformanceIssueEmailView(View):
-    @override_options({"performance.issues.all.problem-creation": 1.0})
```

Comment:
This was just redundant and I missed it in previous cleanups. The same override options are used in the get function.